Files/MultilingualResources/Files.es-ES.xlf

```diff
@@ -256,7 +256,7 @@
         </trans-unit>
         <trans-unit id="ExtractFilesDialogBrowseButton.Content" translate="yes" xml:space="preserve">
           <source>Browse</source>
-          <target state="new">Navegar</target>
+          <target state="new">Examinar</target>
```

Comment:
Unless the state is set to `Translated`, the translation won't show up.

Reply:
oh I didn't know that